Makefile

```diff
@@ -14,8 +18,9 @@ test: all
 
 clean:
 	@rm -rf _book
+	@rm -f book.pdf analysis-essentials.pdf
 	@rm -rf node_modules
 	@rm -f tests/*.pyc
 
-publish-travis: all
+publish-travis: all pdf
 	@ghp-import -n ./_book && git push -fq https://${GH_TOKEN}@github.com/$(TRAVIS_REPO_SLUG).git gh-pages > /dev/null
```

Comment:
I'm not super familiar with it, but I think `ghp-import` will replace the `gh-pages` branch with the contents of the `_book` directory. The PDF made by `make pdf` is `analysis-essentials.pdf` in the _root_ of the repo, so we should `cp` it to `_book` in this rule.

Reply:
I *think* it gets copied by the `otherlink` plugin.

Reply:
OK, let's merge then, and if you're wrong you get the privilege of fixing it 😉

Reply:
[It works!!!!](https://lhcb.github.io/analysis-essentials/analysis-essentials.pdf) 🎉

Reply:
😵